src/form/bf_mx_lin.jl

```diff
@@ -439,21 +440,21 @@ function constraint_mc_load_power(pm::LPUBFDiagModel, load_id::Int; nw::Int=nw_i
         elseif load["model"]==IMPEDANCE
             w = var(pm, nw, :w)[bus_id][[c for c in connections]]
             for (idx,c) in enumerate(connections)
-                JuMP.@constraint(pm.model, pd[idx]==3*a[idx]*w[idx])
-                JuMP.@constraint(pm.model, qd[idx]==3*b[idx]*w[idx])
+                JuMP.@constraint(pm.model, pd[idx]==3*a[idx]*w[c])
+                JuMP.@constraint(pm.model, qd[idx]==3*b[idx]*w[c])
```

Comment:
I'm not sure this is correct, because of how `w` is created in line 441; it should be a vector in the order of connections, and therefore should still be indexed by `idx`

Reply:
I did a quick check by adding `@show pd[idx],qd[idx],a[idx],b[idx],w[c]` inside the for loop and the expressions used for each term are consistent with each other for all three iterations of the loop. If `w[idx]` is used, the order does not seem right. 

Actually, replacing `w = var(pm, nw, :w)[bus_id][[c for c in connections]]` with `w = var(pm, nw, :w)[bus_id]` without changing anything else also keeps the expressions consistent inside the for loop and gives the exact same results!

Reply:
Lets make that change to `w` then, i think it would be better, and should be guaranteed to be consistent in edge cases where the order is unusual.